packages/components/src/Menu/Menu.tsx

```diff
@@ -208,15 +215,30 @@ export interface ActionProps {
    * Callback when an action gets clicked
    */
   onClick?(event: React.MouseEvent<HTMLButtonElement>): void;
+
+  /**
+   * Focus on the action when rendered
+   */
+  shouldFocus?: boolean;
 }
 
-function Action({ label, icon, onClick }: ActionProps) {
+function Action({ label, icon, onClick, shouldFocus = false }: ActionProps) {
+  // eslint-disable-next-line no-null/no-null
```

Comment:
Can you please remove this linking comment?
```suggestion
```

Reply:
Oh, I forgot. Let me quickly do it 👍. Thanks